src/converse-resize-occupants-view.js

```diff
@@ -0,0 +1,161 @@
+// Converse.js (A browser based XMPP chat client)
+// https://conversejs.org
+//
+// Copyright (c) 2012-2017, Jan-Carel Brand <jc@opkode.com>
+// Licensed under the Mozilla Public License (MPLv2)
+//
+/*global define, window, document */
+
+import "converse-chatview";
+import "converse-controlbox";
+import converse from "@converse/headless/converse-core";
+
+const { _ } = converse.env;
+
+converse.plugins.add('converse-resize-occupants-view', {
+    /* Plugin dependencies are other plugins which might be
```

Comment:
Can all this code rather go into `converse-dragresize.js`? They are very similar and both have to do with dragging and resizing elements, and by combining them we don't have to keep track of an extra plugin.

If you do that, then you also don't need to create the extra `converse-mouse-events` plugin either, since all the code can be in one plugin.